src/styles/theme/themeConfig.ts

```diff
@@ -0,0 +1,216 @@
+// Base colors - updates should match css vars!
+const PURE_WHITE = "#ffffff";
+const DARK = "#141219";
+const DARK_TWO = "#1e1b25";
+const DARK_THREE = "#25222e";
+const DARK_FOUR = "#2f2a3c";
+const OFF_WHITE = "#f7f7f7";
+const WHITE_SIX = "#e7e7e7";
+const WHITE_THREE = "#d8d8d8";
+const BABY_PURPLE = "#b59ff6";
+const LIGHT_PURPLE = "#e7e4f2";
+const CHARCOAL_GRAY = "#3b3649";
+const ALLEN_PURPLE = "#8d87aa";
+const GRAYISH_BROWN = "#4a4a4a";
+const BLUE = "#0094FF";
+const WARM_GRAY = "#979797";
+const TRANSPARENT_WHITE = "rgba(255, 255, 255, 0.65)";
+const TRANSPARENT = "transparent";
+
+// Light Theme Tokens (Global defaults, Modal, Radio)
+const lightTheme = {
+    // Global Tokens
+    colorPrimary: BABY_PURPLE,
+    colorBgBase: WHITE_SIX,
+    colorBgContainer: OFF_WHITE,
+    colorBgLayout: WHITE_SIX,
+    colorText: GRAYISH_BROWN,
+    colorLink: BLUE,
+
+    // Form
+    labelColor: DARK,
+
+    // Input
+    inputPlaceholderColor: WARM_GRAY,
+    inputBorderColor: WARM_GRAY,
+    inputBgColor: OFF_WHITE,
+    inputHoverColor: BABY_PURPLE,
```

Comment:
Overall this is nice, but some of the repetition here is more detailed than I think is necessary. For instance, I would have a single light AppBgColor, and then assign it below for both `colorBgContainer` and `inputBgColor` because you're always going to want those two to change together, instead of having a 1:1 of the token names in here. Similarly I would have a single `pageBgColor: WHITE_SIX` and use it for the base and the layout.  I'm also surprised you have to define the inputHoverColor separately, doesn't it already get the primaryColor?